src/app/Main/Body/QueryPage/Results/Charts/QueryPageContent-BarChart.component.tsx

```diff
@@ -18,14 +18,14 @@ const explanationTextStyle = css`
 
 function processingTableDataToApexData(
     data: Record<string, string>[],
-    xLabels: string[]
+    dataLabels: string[]
 ): ApexChartData {
     const apexChartData = {} as ApexChartData;
     const apexChartSeries = [] as typeof apexChartData.series;

```

Comment:
i found a cool new feature: lookup types. So you can do something like:
`const apexChartSeries = [] as ApexChartData['series']`
Then you can remove the const apexChartData and gather the type directly from the interface.

Reply:
Cool! Done.